components/combobox/stories/template.js

```diff
@@ -26,15 +25,9 @@ const Combobox = ({
 	isFocused = false,
 	isKeyboardFocused = false,
 	isLoading = false,
-	selectedDay,
+	value,
 } = {}, context = {}) => {
-	const { globals = {}, updateArgs } = context;
-	const lang = globals.lang ?? "en-US";
-
-	// If selectedDay is a string, convert it to a Date object
-	if (typeof selectedDay === "string" && selectedDay.length > 0) {
-		selectedDay = new Date(selectedDay).toLocaleDateString({ language: lang });
-	}
```

Comment:
I know we talked about this at our dev sync, and I don't totally see a reason for this to be here in combobox. As far as I can tell, this and the subsequent `value: selected Day...` lines are a good removals 🤷‍♀️